### DISCUSSION TITLE
[Question] Is there a way to save TabularModel and use it with other libs?

### DISCUSSION THREAD

**Discussion by siddhantpathakk** [manujosephv/pytorch_tabular#345] [General]
I need to save the model checkpoint trained using TabularModel. I wish to use this model checkpoint later into an external library called "alibi-detect". It is useful for detecting data drift statistically. 

Alibi-detect documentation states it supports loading ML models saved as per the link : https://docs.seldon.io/projects/alibi-detect/en/stable/overview/saving.html#supported-ml-models 

Is there a way to load a TabularModel into Alibi-Detect detector?

**Comment by manujosephv** [comment]
You can save the model as a pure PyTorch model. But you will have to handle the data transformations like normalisation etc on your own by saving the datamodule separately.

#268 has details on how to save models in different formats

**Comment by manujosephv** [comment]
Closing this for now. Feel free to reopen if you feel like something is missing.. or start a discussions thread..
